### PR TITLE
Pull request/f75a5349

### PR DESCRIPTION
**Pull request by montyvesselinov**
Please register:

MADS : Model Analysis and Decision Support
BIGUQ : Bayesian Information Gap Decision Theory for Uncertainty Quantification, Experimental Design and Decision Analysis
Anasol: Analytical solutions for groundwater contaminant transport
MPTools: Tools to evaluate expressions
R3Function: Automated storage and retrieval of results for functions calls

### CODE REVIEW

File: Mads/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,25 @@
+julia 0.4-
```
Comment (tkelman) [inline Mads/versions/0.1.0/requires]: note that the trailing dash refers to prereleases and is no longer necessary if all your users are on 0.4.0 or newer.

**Comment by tkelman** [comment]
MPTools and R3Function are somewhat unclear names, what do the MP and R3 stand for? We try to discourage acronym package names that aren't self evident or well known.

**Comment by montyvesselinov** [comment]
If we rename them "ReusableFunctions.jl" and "MetaProgTools.jl" is it going
to be ok?

-monty

**Comment by tkelman** [comment]
I like those names, that would be a lot clearer IMO.

**Comment by montyvesselinov** [comment]
I made a new pull request of a new version with suggested changes.

-monty

**Comment by tkelman** [comment]
Yep! Merged #4490.

**Comment by montyvesselinov** [comment]
Thank you very much!

-monty

**Pull request closed without merging**